backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/PatientManagementTest.java

```diff
@@ -468,8 +468,8 @@ void updatePatient_invalidArguments_expectedError(
             "patientId", "x".repeat(35), "/updatePatient/patientId size must be between 36 and 36"),
         Arguments.of(
             "firstName",
-            "x".repeat(500),
-            "/updatePatient/firstName size must be between 0 and 256"));
+            "x".repeat(5000),
+            "/updatePatient/firstName size must be between 0 and 4096"));
```

Comment:
Have we considered what knock-on effects increasing this directive size may have, from a security standpoint or otherwise? It seems like this just increases the size for all arguments, and by a factor of almost 20. 

In addition, have we done some testing on this in a live environment to ensure existing functionality is not compromised?

Reply:
this filter is not just related to this query, its for every query in graphql, and its just increasing the size of the input object, you still have to be authenticated. do you have any particular concerns?

Reply:
Right, my concern is primarily that this will end up touching every single query, and not just the one that spurs this change. It's likely innocuous, but it seems like we're using a wrecking ball for what would be better suited with a very tiny hammer. I understand that authentication is required to get this to return anything, but longer upward limits have historically been used to facilitate web server attacks. I'm asking if this fundamental change will spur any security concerns, given that it will potentially impact everything.

In addition, please reference my comment RE: testing. Has this been rolled out to a live environment and smoke tested on a range of queries, to catch any weird behavior?

Reply:
yeah I hear you, its on test and everything seems to be working fine, is there any specific concerns? since you can only hit any graphql endpoint when you are authenticated, and this is just a filter on the query variables size

Reply:
Great, thanks for smoke testing that. 

I'm a GraphQL newbie, so I'm still learning how it works, but I'm always concerned when we make a change that seems to affect everything. My general concerns are listed in my previous comment. Yes, I understand you can only hit the endpoint when authenticated, but since this affects all of GraphQL, I want to make sure it's airtight. Could this theoretically allow someone, accidentally or otherwise, to submit a 4096-character-long last name, city, street address, or otherwise, that we aren't expecting? I'm sure we have UI validation in place, but that's trivial to bypass with a determined user, which is why I like seeing server-side checks in place for unexpected lengths. Do we have those in place elsewhere?

Ultimately, I'm trying to make sure there isn't a "right" way to do this that is different from what you have already done here. Is there a way to change the filter for just the "resend" endpoint you mentioned in the description of this PR?

Reply:
there isn't any obvious way to just limit this to the only mutation that I need, but we can put this back after addressing all the RS resend requests that came from the incident on 1/30

Reply:
Next time, let's chat about this before a merge. I understand that this was an urgent request from RS, but we can't compromise our quality or security for the sake of someone else's convenience. I would rather take extra time to build in the endpoint-specific changes, which would give us the added benefit of being able to leverage that new functionality in the future.

I'm going to create a reversion PR for this, just so it doesn't fall off our radar. I'll make sure it doesn't get merged until those resends are complete.